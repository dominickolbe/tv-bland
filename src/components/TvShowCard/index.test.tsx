import { shallow } from "enzyme";
import React from "react";
import tvshowJSON from "../../__mock__/tvshow.json";
import { TvShowCard } from "./";

describe("<TvShowCard />", () => {
  it("renders snapshots", () => {
    const wrapper = shallow(
      <TvShowCard tvshow={tvshowJSON} onClick={jest.fn()} />
    );
    expect(wrapper).toMatchSnapshot();
  });
});
