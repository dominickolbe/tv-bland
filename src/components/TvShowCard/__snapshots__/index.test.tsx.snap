// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TvShowCard /> renders snapshots 1`] = `
<WithStyles(ForwardRef(Card))>
  <WithStyles(ForwardRef(CardActionArea))
    onClick={[Function]}
  >
    <WithStyles(ForwardRef(CardMedia))
      className="makeStyles-cover-1"
      image="https://static.tvmaze.com/uploads/images/medium_portrait/173/433868.jpg"
      title="The Big Bang Theory"
    />
  </WithStyles(ForwardRef(CardActionArea))>
  <WithStyles(ForwardRef(CardContent))>
    <div
      className="makeStyles-rating-2"
    >
      <Rating
        value={7.9}
        withText={false}
      />
    </div>
    <WithStyles(ForwardRef(Typography))
      color="textSecondary"
      component="div"
      variant="body2"
    >
      <Styled(MuiBox)>
        The Big Bang Theory
      </Styled(MuiBox)>
    </WithStyles(ForwardRef(Typography))>
  </WithStyles(ForwardRef(CardContent))>
</WithStyles(ForwardRef(Card))>
`;
